kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -1580,4 +1600,52 @@ private void makeNewPatternWindow(ObservableViewNoOverride windowView, NidTextEn
         //FIXME opening the panel too soon creates a broken UI for the pattern window
         //EvtBusFactory.getDefaultEvtBus().publish(patternViewModel.getPropertyValue(PATTERN_TOPIC), new PropertyPanelEvent(patternConfig, OPEN_PANEL));
     }
+
+
+    private void makePatternWindow(EntityFacade patternFacade, ViewProperties viewProperties) {
+        //FIXME need purpose and meaning of the pattern, the STAMP values, the descriptions, the fields, the name and public id
+        Entity patternEntity = EntityService.get().getEntity(patternFacade.nid()).get();
+
+        // populate STAMP values
+        Latest<EntityVersion> patternStamp = viewProperties.calculator().stampCalculator().latest(patternEntity);
+
+        StampViewModel stampViewModel = new StampViewModel();
+        stampViewModel.setPropertyValue(STATUS, patternStamp.get().stamp().state())
+            .setPropertyValue(TIME, patternStamp.get().stamp().time())
+            .setPropertyValue(AUTHOR, patternStamp.get().stamp().author())
+            .setPropertyValue(MODULE, patternStamp.get().stamp().module())
+            .setPropertyValue(PATH, patternStamp.get().stamp().path())
+            ;
+
+        stampViewModel.setPropertyValue(PATHS_PROPERTY, stampViewModel.findAllPaths(viewProperties), true)
+                .setPropertyValue(MODULES_PROPERTY, stampViewModel.findAllModules(viewProperties), true);
+
+        //TODO add 'edit' states to the state machine
+        StateMachine patternSM = StateMachine.create(new PatternDetailsPattern());
+
+        ValidationViewModel patternViewModel = new PatternViewModel()
+                .setPropertyValue(VIEW_PROPERTIES, viewProperties)
+                .setPropertyValue(MODE, EDIT)
+                .setPropertyValue(STAMP_VIEW_MODEL, stampViewModel)
+                .setPropertyValue(PATTERN_TOPIC, UUID.randomUUID())
+                .setPropertyValue(STATE_MACHINE, patternSM)
+                .setPropertyValue(PATTERN, patternFacade)
+                ;
+        Config patternConfig = new Config(PatternDetailsController.class.getResource("pattern-details.fxml"))
+                .addNamedViewModel(new NamedVm("patternViewModel", patternViewModel));
+
+        // create pattern window
+        JFXNode<Pane, PatternDetailsController> patternJFXNode = FXMLMvvmLoader.make(patternConfig);
+        patternJFXNode.controller().updateView();
```

Comment:
Can we remove the `updateView()` method?
Reasoning: Once the updateViewModel() method will update data (properties).

Reply:
the entire logic for populating the window is currently in updateView() ... we should get on a call and discuss.